plugins/doc_fragments/assume_role.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2015, Ansible, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+class ModuleDocFragment:
+    # Note: If you're updating MODULES, PLUGINS probably needs updating too.
+
+    # Formatted for Modules
+    # - modules don't support 'env'
+    MODULES = r"""
+options: {}
+"""
+
+    # Formatted for non-module plugins
```

Comment:
Is it worth adding a note in the README about this doc_fragment?

Reply:
The scope is currently limited to inventory plugins, so I wasn't going to add one just yet.